Bug: [wr_arp] The test case arp/test_wr_arp.py::test_wr_arp is not aligned with change in ferret.py(PR#15835)

### Is it platform specific

generic

### Importance or Severity

Critical

### Description of the bug

There is a change in the tests/arp/files/ferret.py made by PR#15835.
It changed the padding bytes in the payload of the arp packets:
https://github.com/sonic-net/sonic-mgmt/pull/15835/files#diff-51cd7fcc6e0d30d81b0254bafba60849e0e26e9f308e5779d1f57f3d46a279beR177

But the test arp/test_wr_arp.py::test_wr_arp is not aligned with this change, the arp packets sent by the ferret will not match the expected packet in the test.
This happens in testPort function of ansible/rolse/test/files/ptftests/py3/wr_arp.py when comparing with count_matched_packets.

This causes failure like this:
```
E           2025-02-24 23:54:39 : Port eth9. Max pause in arp_response 38 sec
E           2025-02-24 23:54:39 : Port eth10. Max pause in arp_response 38 sec
E           2025-02-24 23:54:39 : Port eth11. Max pause in arp_response 38 sec
E           2025-02-24 23:54:39 : Port eth12. Max pause in arp_response 38 sec
E           2025-02-24 23:54:39 : Port eth13. Max pause in arp_response 38 sec
E           2025-02-24 23:54:39 : Port eth14. Max pause in arp_response 38 sec
E           2025-02-24 23:54:39 : Port eth15. Max pause in arp_response 38 sec
E           2025-02-24 23:54:39 : Port eth22. Max pause in arp_response 38 sec
E           2025-02-24 23:54:39 : Port eth23. Max pause in arp_response 38 sec
E           2025-02-24 23:54:39 : Port eth24. Max pause in arp_response 38 sec
E           
E           
E           ******************************************
E           ATTENTION: SOME TESTS DID NOT PASS!!!
E           
E           The following tests failed:
E           ArpTest
E           
E           ******************************************stderr =
E           /root/env-python3/lib/python3.7/site-packages/scapy/layers/ipsec.py:512: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.
E             cipher=algorithms.TripleDES,
E           /root/env-python3/lib/python3.7/site-packages/scapy/layers/ipsec.py:516: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.
E             cipher=algorithms.TripleDES,
E           wr_arp.ArpTest ... 23:52:17.725  device_connection: ERROR   : Caught exception socket.timeout: timeout(), , <class 'socket.timeout'>
E           FAIL
E           
E           ======================================================================
E           FAIL: wr_arp.ArpTest
E           ----------------------------------------------------------------------
E           Traceback (most recent call last):
E             File "ptftests/py3/wr_arp.py", line 304, in runTest
E               self.test_wrarp_basic()
E             File "ptftests/py3/wr_arp.py", line 384, in test_wrarp_basic
E               self.assertTrue(len(m_pauses) == 0, "Too long pauses in arp responses")
E           AssertionError: False is not true : Too long pauses in arp responses
E           
E           ----------------------------------------------------------------------
E           Ran 1 test in 371.842s
E           
E           FAILED (failures=1)
```

### Steps to Reproduce

Run the test arp/test_wr_arp.py::test_wr_arp on any platforms.

### Actual Behavior and Expected Behavior

Test should pass.

### Relevant log output

```shell

```

### Output of `show version`

```shell

```

### Attach files (if any)

_No response_

--- a/ansible/roles/test/files/ptftests/py3/wr_arp.py
+++ b/ansible/roles/test/files/ptftests/py3/wr_arp.py
@@ -176,6 +176,7 @@ class ArpTest(BaseTest):
         )
 
         exp_pkt = testutils.simple_arp_packet(
+            pktlen=42,
             ip_snd=gw,
             ip_tgt=port_ip,
             eth_src=self.dut_mac,

